components/Navbar.jsx

```diff
@@ -19,7 +19,7 @@ function Navbar({ router }) {
 
         <ul className="nav-links">
           {navs.map((nav) => (
-            <li data-cy={nav.dataCy} key={`nav-${nav.href}`}>
+            <li data-test-id={nav.dataCy} key={`nav-${nav.href}`}>

```

Comment:
Change the `data-cy` attribute from Cypress to `data-test-id` attribute from Playwright